src/atomics/fences.md

```diff
@@ -0,0 +1,257 @@
+# Fences
+
+As well as loads, stores, and RMWs, there is one more kind of atomic operation
+to be aware of: fences. Fences can be triggered by the
+[`core::sync::atomic::fence`] function, which accepts a single ordering
+parameter and returns nothing. They don’t do anything on their own, but can be
+thought of as events that strengthen the ordering of nearby atomic operations.
```

Comment:
On first read, I thought "ordering" as of "re-ordering" but now I understand that it means memory ordering. I think it would be very wise to make sure readers understand the difference and cannot confuse the two. I would propose to use ordering in a sense memory-ordering always as `ordering`